tests/xfstests/run.pm

```diff
@@ -302,7 +306,7 @@ sub run {
         select_console('root-console');
         # Save kdump data to KDUMP_DIR if not set "NO_KDUMP"
         unless (get_var('NO_KDUMP')) {
-            unless (save_kdump($test, $KDUMP_DIR, "vmcore", "kernel")) {
+            unless (save_kdump($test, $KDUMP_DIR, ('vmcore', 1, 'kernel', 1, 'debug', 1))) {
```

Comment:
ahh , I was not aware that you can pass hash like this . anyway I think this is more readable way : 
```suggestion
            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1)) {
```

Reply:
and don't try to use github magic to apply changes :) I think it will not fit our flows ...

Reply:
Thank you for review, just modify it :)